devices/debian.py

```diff
@@ -351,11 +351,21 @@ def setup_as_cmts_provisioner(self, board_config):
         self.expect(self.prompt)
         self.sendline('ifconfig eth1 %s' % self.gw)
         self.expect(self.prompt)
+        self.sendline('ifconfig eth1 inet6 add 2001:ed8:77b5:3::101/64')
+        self.expect(self.prompt)
         # TODO: specify these via config
         self.sendline('ip route add 192.168.201.0/24 via 192.168.3.222')
         self.expect(self.prompt)
         self.sendline('ip route add 192.168.200.0/24 via 192.168.3.222')
         self.expect(self.prompt)
+        self.sendline('ip -6 route add 2001:ed8:77b5:2000::/64 via 2001:ed8:77b5:3::222 dev eth1  metric 1024')
+        self.expect(self.prompt)
+        self.sendline('ip -6 route add 2001:ed8:77b5:2001::/64 via 2001:ed8:77b5:3::222 dev eth1  metric 1024')
+        self.expect(self.prompt)
+        #check if 'dadfailed' happen  
+        self.sendline('ip addr show eth1')
```

Comment:
You should really check the output with expect and look for the 'dadfailed' - at least print a warning if we see it but should probably fail the test. It seems like we are broken in this state.

Reply:
I did see the dadfailed if I add ipv6 address to eth1 manually which we don't know the reason yet. But with adding the ipv6 address via boardfarm code I don't see it happens during many times of testing.

Reply:
You can add a 
```
if 0 == self.expect(['dadfailed'] + prompt):
    raise Exception("ipv6 dadfailed present, ipv6 will not funtion")
```